src/components/pages/Oppsummering/Innrapportering.jsx

```diff
@@ -21,7 +21,8 @@ const Innrapportering = withBehandlingContext(({ behandling }) => {
             txt: JSON.stringify(
                 innrapportering.uenigheter.map(uenighet => ({
                     label: uenighet.label,
-                    value: uenighet.value
+                    value: uenighet.value,
+                    userId: uenighet.userId
```

Comment:
Trenger vi ident for hver uenighet, kontra å sette det på hele innrapporteringen?

Reply:
Sånn jeg har skjønt det er det mulig at flere saksbehandlere kan være innom samme behandling og dermed komme med tilbakemeldinger på forskjellige felter. Derfor kan det være greit å skille mellom hvem som har sagt hva. Tenkte også først at det var like greit å ha én ident for hele rapporten.

Reply:
Ja, det gir kanskje mening, det. Men nå ser det ut til at siste"mann" som sender inn feedback vil få sitt navn på alle uenigheter?

Reply:
Sånn det ser ut nå får ikke saksbehandlere se hva andre saksbehandlere har vært uenige i. Når de går inn på en behandling får de mao. "blanke ark" siden tidligere uenigheter ikke sendes fra BE. Det er kanskje like greit? Men da må hele rapporten sendes med saksbehandleres ident + behandlingsid som id for at s3 ikke skal overskrive tidligere rapporter.

Reply:
Jeg ville ha trodd det var greit å se at en annen saksbehandler har vært innpå, men det er sikkert akseptabelt som det er i første omgang, og så får vi ta en prat med Nina neste gang hun er her :)